semantic-conventions/README.md

```diff
@@ -59,6 +80,106 @@ const span = tracer.startSpan(spanName, spanOptions)
+
+Deprecated as of `@opentelemetry/semantic-conventions@1.26.0`.
+
+Before v1.26.0, constants for each semconv attribute were exported, prefixed with `SEMRESATTRS_` (if defined as a Resource Attribute) or `SEMATTRS_`. As well, constants were exported for each value in an enumeration, of the form `${attributeName}VALUES_${enumValue}`. For example:
+
+**Deprecated usage:**
+
+```js
+import {
+  SEMRESATTRS_SERVICE_NAME,
+  SEMATTRS_HTTP_ROUTE,
+  SEMATTRS_DB_SYSTEM,
+  DBSYSTEMVALUES_POSTGRESQL
+} from '@opentelemetry/semantic-conventions';
+
+// 'service.name' resource attribute
+console.log(SEMRESATTRS_SERVICE_NAME); // migrate to 'ATTR_SERVICE_NAME'
+
+// 'http.route' attribute
+console.log(SEMATTRS_HTTP_ROUTE); // migrate to 'ATTR_HTTP_ROUTE'
+
+// 'db.system' attribute
+console.log(SEMATTRS_DB_SYSTEM); // migrate to 'ATTR_DB_SYSTEM' (in incubating [*])
+
+// 'postgresql' enum value for 'db.system' attribute
+console.log(DBSYSTEMVALUES_POSTGRESQL); // migrate to 'DB_SYSTEM_VALUE_POSTGRESQL' (in incubating [*])
+```
+
+See [Migrated usage](#migrated-usage) below.
+
+### Migrating from `SemanticAttributes.*`, `SemanticResourceAttributes.*`, ...
+
+Deprecated as of `@opentelemetry/semantic-conventions@1.0.0`.
+
+Before v1.0.0, constants were exported in namespace objects `SemanticResourceAttributes` and `SemanticAttributes`, and a namespace object for enumerated values for some fields (e.g. `DbSystemValues` for values of the 'db.system' enum). For example:
+
+**Deprecated usage:**
+
+```js
+import {
+  SemanticAttributes,
+  SemanticResourceAttributes,
+  DbSystemValues,
+} from '@opentelemetry/semantic-conventions';
+
+// 'service.name' resource attribute
+console.log(SemanticResourceAttributes.SERVICE_NAME); // migrate to 'ATTR_SERVICE_NAME'
+
+// 'http.route' attribute
+console.log(SemanticAttributes.HTTP_ROUTE); // migrate to 'ATTR_HTTP_ROUTE'
+
+// 'db.system' attribute
+console.log(SemanticAttributes.DB_SYSTEM); // migrate to 'ATTR_DB_SYSTEM' (in incubating [*])
+
+// 'postgresql' enum value for 'db.system' attribute
+console.log(DbSystemValues.POSTGRESQL); // migrate to 'DB_SYSTEM_VALUE_POSTGRESQL' (in incubating [*])
+```
+
+See [Migrated usage](#migrated-usage) below.
+
+### Migrated usage
+
+```js
+import {
+  ATTR_SERVICE_NAME,
+  ATTR_HTTP_ROUTE,
+  METRIC_HTTP_CLIENT_REQUEST_DURATION
+} from '@opentelemetry/semantic-conventions';
+import {
+  ATTR_DB_SYSTEM,
+  DB_SYSTEM_VALUE_POSTGRESQL
+} from '@opentelemetry/semantic-conventions/incubating';
+
+console.log(ATTR_SERVICE_NAME); // 'service.name'
+console.log(ATTR_HTTP_ROUTE);   // 'http.route'
+
+// Bonus: the older exports did not include metric names from semconv.
+// 'http.client.request.duration' metric name
+console.log(METRIC_HTTP_CLIENT_REQUEST_DURATION);
+
+console.log(ATTR_DB_SYSTEM);    // 'db.system'
+// 'postgresql' enum value for 'db.system' attribute
+console.log(DB_SYSTEM_VALUE_POSTGRESQL);
+```
+
+**What is "incubating"?**
```

Comment:
```suggestion
**What is "incubating"?**

```
Generally would prefer a header here (`####`) and in other bolded sections but total nit

Reply:
handled in commit 8fedc235e, at least for this one. I didn't really want a *section* for the bolded "Deprecated usage." cases above.